src/current/v23.2/debezium.md

```diff
@@ -56,32 +56,60 @@ Once all of the [prerequisite steps](#before-you-begin) are completed, you can u
+        "database.port": "5432",
+        "database.user": "postgres",
+        "plugin.name": "pgoutput",
+        "table.include.list": "public.test_table_small",
+        "tasks.max": "1",
+        "topic.creation.default.cleanup.policy": "delete",
+        "topic.creation.default.partitions": "10",
+        "topic.creation.default.replication.factor": "1",
+        "topic.creation.default.retention.ms": "604800000",
+        "topic.creation.enable": "true",
+        "topic.prefix": "{username}",
+        "slot.name" : "debezium"
+      }
+    }
+    ~~~
+
 1. Create the JSON configuration file that you will use to create the sink:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
     {
-       "name": "pg-sink",
-       "config": {
-           "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector", 
-           "tasks.max": "10",
-           "topics" : "{topic.example.table}",
-           "connection.url": "jdbc:postgresql://{host}:{port}/{user}?sslmode=require",
-           "connection.user": "{username}",
-           "connection.password": "{password}",
-           "insert.mode": "upsert",
-           "pk.mode": "record_value",
-           "pk.fields":"id",
-           "database.time_zone": "UTC",
-           "auto.create":true,
-           "auto.evolve": false,
-           "transforms": "unwrap",
-           "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
-       }
+      "name": "pg-sink",
+      "config": {
+        "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector", 
+        "tasks.max": "10",
+        "topics" : "{topic.example.table}",
+        "connection.url": "jdbc:postgresql://{host}:{port}/{username}?sslmode=require",
+        "connection.user": "{username}",
+        "connection.password": "{password}",
+        "insert.mode": "{insertion mode}",
+        "pk.mode": "record_value",
+        "pk.fields": "id",
+        "database.time_zone": "UTC",
+        "auto.create": true,
+        "auto.evolve": false,
+        "transforms": "unwrap",
+        "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
+      }
     }
     ~~~
     
-    Specify the **Connection URL** in [JDBC format]({% link {{ page.version.version }}/connect-to-the-database.md %}?filters=java&#step-5-connect-to-the-cluster). For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster]({% link {{ page.version.version }}/connect-to-the-database.md %}).
+    - Specify `connection.url` in [JDBC format]({% link {{ page.version.version }}/connect-to-the-database.md %}?filters=java&#step-5-connect-to-the-cluster). For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster]({% link {{ page.version.version }}/connect-to-the-database.md %}).
+    - If you set `insert.mode` to `upsert`, you must set `pk.mode` and `pk.fields`. Set `pk.fields` to your primary key.
```

Comment:
I don't actually recall if this statement is true.... I think maybe if we just link the docs of possible connector options is good enough and we call out everything here is just an example and modify to your needs?

Reply:
Updated, PTAL!